doc/whats_new/v1.1.rst

```diff
@@ -60,7 +60,7 @@ Changelog
   add this information to the plot.
   :pr:`21038` by :user:`Guillaume Lemaitre <glemaitre>`.
 
-- |Enhancement| :class:`cluster.SpectralClustering` and :func:`cluster.spectral` 
+- |Enhancement| :class:`cluster.SpectralClustering` and :func:`cluster.spectral`
```

Comment:
```suggestion
- |Enhancement| :class:`cluster.SpectralClustering` and :func:`cluster.spectral` 
```

Reply:
We have trailing spaces removal in the pre-commit config. It would force me to not run the pre-commit (and thus all the `black` and `flake8` stuff). I assume it is better to just correct it here.